### PR TITLE
Add Navigation to Install

### PR DESCRIPTION
**Pull request by DLu**

### CODE REVIEW

File: euler.rosinstall [hunk 1]
```diff
@@ -7,3 +7,6 @@
     version: euler_integration}
 - git: {local-name: sicktoolbox_wrapper, uri: 'https://github.com/swri-robotics/sicktoolbox_wrapper.git',
     version: euler_integration}
+- git: {local-name: navigation, uri: 'https://github.com/DLu/navigation.git',
```
Comment (shaun-edwards) [inline euler.rosinstall]: @DLu, do you mind if we fork your repo to swri-robotics.  This way you won't have to worry about/remember to keep that branch around.  Also, do you plan on rolling in "our" changes into the main navigation repo (It's OK if not).

Comment (DLu) [inline euler.rosinstall]: That sounds like a good plan. The changes will eventually be rolled into the main navigation repo, once we create an indigo branch.

Comment (DLu) [inline euler.rosinstall]: Let me know when fork it and I'll edit this pull request to point to it.

**Comment by shaun-edwards** [comment]
Forked simple local planner: https://github.com/swri-robotics/simple_local_planner

**Comment by DLu** [comment]
Do you want to fork the main navigation repo too, or wait until there's an indigo branch?

**Comment by shaun-edwards** [comment]
Forked navigation: https://github.com/swri-robotics/navigation

**Comment by DLu** [comment]
Updated with new fork.

**Comment by DLu** [comment]
Also, I've created a pull request to get the navigation changes into the main indigo branch. 

https://github.com/ros-planning/navigation/pull/304

**Pull request closed without merging**